tests/README.md

```diff
@@ -16,3 +16,24 @@ In the text input box in Mattermost, type: `/test url [file-name-in-testing-fold
 
 ## Manual Testing  
 It is possible to manually test specific sections of any test, instead of using the /test command. Do this by clicking **Raw** in the header for the file when it’s open in GitHub, then copy and paste any section into Mattermost to post it. Manual testing only supports sections of 4000 characters or less per post.
+
+## Test plugins
+
+There are two test plugins: `testplugin.tar.gz` and `testpluginv2.tar.gz`. These are use in some integration tests in the `api4` package. Any changes to the plugin bundles require updating the coresponding signatures.
```

Comment:
As @marianunez also notes below, this changed with the merge of the prepackaged rework to remove confusion about the "version" of the plugin:
```suggestion
There are two test plugins: `testplugin.tar.gz` and `testplugin2.tar.gz`. These are use in some integration tests in the `api4` package. Any changes to the plugin bundles require updating the coresponding signatures.
```